src/index.js

```diff
@@ -41,117 +42,125 @@ const App = () => {
     <Request url={decodedToken.user_url}>
       {({data}) => {
 
-        if(data && !data.instructor_url) {
+        const user = data
```

Comment:
I renamed the request data to be more explicit and avoid confusion with inner requests